lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/components/sidebar.scss

```diff
@@ -0,0 +1,37 @@
+.sidebar {
+  display: flex;
+  overflow: auto;
+  flex: 0 0 250px;
+  flex-direction: column;
+
+  width: 100%;
+  padding: 0 var(--space-md) var(--space-lg);
+
+  background: var(--color-white);
+  box-shadow: var(--shadow);
+
+  a {
+    text-decoration: none;
+  }
+}
+
+.sidebar-nav__item {
+  margin-bottom: 1rem;
```

Comment:
This should use a `var`

Reply:
Ideally we would not use a margin here and instead use gap on a parent element.